### PR TITLE
Add ConversionsParametersStudy to control default patterns for VAC

### PR DESCRIPTION
**Pull request by moritzhaller**

### CODE REVIEW

**Review by kjozwiak — APPROVED** [review]
@iambrianfung code looks good to me but I'll wait till you give it a glance just to double check before merging 👍 Approved by @brave/uplift-approvers.

**Review by iambrianfung — APPROVED** [review]
++

**Pull request merged**
